src/RBACRules.ts

```diff
@@ -31,7 +30,7 @@ const RBACRules: RBACConfig = {
     groupRules: {
         practitioner: {
             operations: ['create', 'read', 'update', 'delete', 'vread', 'search-type', 'transaction'],
-            resources: SUPPORTED_R4_RESOURCES,
+            resources: BASE_R4_RESOURCES,

```

Comment:
Sadly this is an existing bug -- but if you look at the config.ts it could be STU3 making this ruleset nonsensical. Could you close that gap by just creating a constructor that either takes in the version or probably better a list of resources?

Reply:
Updated